google_nest_sdm/webrtc_util.py

```diff
@@ -1,114 +1,108 @@
-# SDP Direction Constants
-DIRECTION_SENDRECV = 'sendrecv'
-DIRECTION_SENDONLY = 'sendonly'
-DIRECTION_RECVONLY = 'recvonly'
-DIRECTION_INACTIVE = 'inactive'
+"""Library with functions for manipulating WebRTC requests/responses."""
+from enum import StrEnum
+
+# SDP direction constants
+class SDPDirection(StrEnum):
+    SENDRECV = "sendrecv"
+    SENDONLY = "sendonly"
+    RECVONLY = "recvonly"
+    INACTIVE = "inactive"
+
+# SDP media kind constants
+class SDPMediaKind(StrEnum):
+    AUDIO = "audio"
+    VIDEO = "video"
+    APPLICATION = "application"
+
+def _get_media_direction(sdp: str, kind: SDPMediaKind) -> SDPDirection | None:
+    """
+    Retrieves the direction of media tracks from the SDP based on the kind (audio/video).
+
+    Args:
+        sdp (str): The SDP content
+        kind (SDPMediaKind): The kind of media track to check ('audio' or 'video').
```

Comment:
Args and Returns are no longer needed given the enum usage is self documenting.  Pydoc should start on the first line.

```suggestion
    """Retrieves the direction of media tracks from the SDP based on the kind (audio/video)."""
```